examples/redis/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.1"
+services:
+  # Redis instance (the thing we're instrumenting)
+  redis_db:
+    container_name: redis_db
+    image: redis:latest
+    command: redis-server /etc/redis/redis.conf
+    volumes:
+      - "./redis.conf:/etc/redis/redis.conf"
+    ports:
+      - "6379:6379"
+      - "6329:6329"
+        #splunk:
+        #  image: splunk/splunk:latest
+        #  container_name: splunk_redis
+        #  environment:
+        #    - SPLUNK_START_ARGS=--accept-license
+        #    - SPLUNK_HEC_TOKEN=00000000-0000-0000-0000-0000000000000
+        #    - SPLUNK_PASSWORD=changeme
+        #  ports:
+        #    - 18000:8000
+        #  healthcheck:
+        #    test: ['CMD', 'curl', '-f', 'http://localhost:8000']
+        #    interval: 5s
+        #    timeout: 5s
+        #    retries: 20
+        #  volumes:
+        #    - ./splunk.yml:/tmp/defaults/default.yml
+        #    - /opt/splunk/var
+        #    - /opt/splunk/etc
+  otelcollector:
+    # if you want to run with the currently released image,
+    #image:  quay.io/signalfx/splunk-otel-collector:latest
+    # if building locally via `make docker-otel`,
+    image:  otelcol:latest
+    container_name: otelcollector-redis-receiver-example
+    environment:
+      - SPLUNK_ACCESS_TOKEN=${SPLUNK_ACCESS_TOKEN}
+      - SPLUNK_REALM=${SPLUNK_REALM}
+      - SPLUNK_PASSWORD=changeme
```

Comment:
this is probably not needed here. Did you mean to spin up a Splunk Enterprise container?

Reply:
Make the choice you want. Both options are ok. Pick the one you can land now.

Reply:
```suggestion
```
You can remove this line.